setup.py

```diff
@@ -190,7 +190,7 @@ def run_tests(self):
             'osc2midi = osc2midi.main:main [osc2midi]',
         ]
     },
-    tests_require=['pytest'],
+    tests_require=['pytest','mock'],
```

Comment:
Okay, here's why I put no space after the comma:

```
setup(
    packages = ['rtmidi', 'osc2midi'],
    package_dir = {'osc2midi': 'examples/osc2midi'},
    ext_modules = cythonize(extensions),
    extras_require = {
        'osc2midi':  ['pyliblo', 'PyYAML'],
    },
    entry_points = {
        'console_scripts': [
            'osc2midi = osc2midi.main:main [osc2midi]',
        ]
    },
    tests_require=['pytest','mock'],
    # On systems without a RTC (e.g. Raspberry Pi), system time will be the
    # Unix epoch when booted without network connection, which makes zip fail,
    # because it does not support dates < 1980-01-01.
    zip_safe=False,
    **setup_opts
)
```

Spaces seem to be in short supply after `entry_points`. Had `tests_require` and `zip_safe` had spaces around `=` like the rest of the parameters, I would have been more careful about adding my own. What's the deal here?